tests/robot/OBSERVATION/01_create.robot

```diff
@@ -797,4 +797,645 @@ Force Tags              observation_create    create
 
     ehr.create new ehr    000_ehr_status.json
     observation.create observation lab aspartate aminotransferase  Aspartate Aminotransferase  create-observation-lab-aspartate-aminotransferase.json
-    observation.validate response - 201	
+    observation.validate response - 201
+
+
+060 Create Observation Blood Pressure loinc datetime
+	[Documentation]     1. *CREATE* new EHR record\n\n 
+	...                 2. *LOAD* _create-blood-pressure_loinc-datetime.json_\n\n
+	...                 3. *UPDATE* ``Subject - Identifier - value`` with the _UUID:_ ${subject_id} which was created in EHR record\n\n
+    ...                 4. *POST* example JSON to observation endpoint\n\n
+	...                 5. *VALIDATE* the response status
+	[Tags]             	blood-pressure    valid    alternative    loinc    datetime
+
+    ehr.create new ehr    000_ehr_status.json
+    observation.create blood pressure  Blood Pressure  create-blood-pressure_loinc-datetime.json
+    observation.validate response - 201
+
+
+061 Create Observation Blood Pressure loinc instant
+	[Documentation]     1. *CREATE* new EHR record\n\n 
+	...                 2. *LOAD* _create-blood-pressure_loinc-instant.json_\n\n
+	...                 3. *UPDATE* ``Subject - Identifier - value`` with the _UUID:_ ${subject_id} which was created in EHR record\n\n
+    ...                 4. *POST* example JSON to observation endpoint\n\n
+	...                 5. *VALIDATE* the response status
+	[Tags]             	blood-pressure    valid    alternative   loinc    instant    not-ready_bug    not-ready
```

Comment:
every test that has a `not-ready_bug` tag should have also a related Github issue. The issue number should be added to the tags